python/ray/tune/trainable/function_trainable.py

```diff
@@ -620,6 +621,15 @@ def wrap_function(
                 "arguments to be `func(config, checkpoint_dir=None)`."
             )
 
+    if use_checkpoint:
+        if log_once("tune_checkpoint_dir_deprecation") and warn:
+            warnings.warn(
+                "`checkpoint_dir` in `func(config, checkpoint_dir)` is being "
+                "deprecated. To save and load checkpoint in tune function, "
+                "please use `ray.air.session` API.",
```

Comment:
Can we make it easier for users to migrate? E.g.

```suggestion
                "deprecated. To save and load checkpoints in trainable functions, "
                "please use the `ray.air.session` API:\n\n",
                "from ray.air import session\n\n",
                "def train(config):\n",
                "    # ...\n",
                "    session.report({"metric": metric}, checkpoint=checkpoint)\n\n",
                "For more information, see https://docs.ray.io/en/master/ray-air/key-concepts.html#session"
```

Reply:
(or without code example and just link to docs)

Reply:
ah good point, added!! Thank you ;)